pandas/tests/series/indexing/test_indexing.py

```diff
@@ -653,6 +653,15 @@ def test_timedelta_assignment():
     expected.loc[[1, 2, 3]] = pd.Timedelta(np.timedelta64(20, "m"))
     tm.assert_series_equal(s, expected)
```

Comment:
should really put this in a test_timedelta.py (and take the existing tests out of test_indexing).

Reply:
after the current batch of PRs I'm planning on doing a review of the indexing tests.  There are multiple dimensions along which we can sort/parametrize, any of which would be reasonable, but my guess is we are not being consistent about it.